getting-started/traced-example/tracing.js

```diff
@@ -19,4 +20,9 @@ provider.addSpanProcessor(
 );
 
 provider.register();
+
+registerInstrumentations({
```

Comment:
Why call `registerInstrumentations()` here without any instrumentation?

Reply:
to load all default old plugins as in old plugin loader.

Reply:
Maybe add a comment that this is just for legacy reasons and only needed if old style plugins are used?